core/src/js/services/mercenaries/highlights/mercenaries-synergies-highlight-processor.ts

```diff
@@ -18,7 +18,7 @@ export const buildSelector = (cardId: string, allCards: CardsFacadeService): Hig
 		case CardIds.ArcaneBlast3Lettuce:
 		case CardIds.ArcaneBlast4Lettuce:
 		case CardIds.ArcaneBlast5Lettuce:
-			return arcane;
+			return arcaneSpellPower;
```

Comment:
Still not perfect (Arcane Blast itself has the SPELLPOWER_ARCANE as a referencedTag, even though it doesn't grant any arcane damage), but probably better than the current implementation